src/tasks/che.ts

```diff
@@ -554,4 +555,42 @@ export class CheTasks {
       }
     ]
   }
+
+  debugTask(flags: any): ReadonlyArray<Listr.ListrTask> {
+    return [
+      {
+        title: 'Find Che Server pod',
+        task: async (ctx: any, task: any) => {
+          const chePods = await this.kube.listNamespacedPod(flags.chenamespace, undefined, this.cheSelector)
+          if (chePods.items.length === 0) {
+            throw new Error(`Che Server pod not found in the namespace '${flags.chenamespace}'`)
+          }
+          ctx.podName = chePods.items[0].metadata!.name!
+          task.title = `${task.title}...done`
+        }
+      },
+      {
+        title: 'Check if debug mode is enabled',
+        task: async (ctx: any, task: any) => {
+          const res = await execa(`kubectl exec ${ctx.podName} -n ${flags.chenamespace} -- printenv CHE_DEBUG_SERVER `, { timeout: 10000, reject: false, shell: true })
```

Comment:
I guess we can get this information from config map.

Reply:
Or you can get env variable from pods container: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#pods-inject-envars-yaml . Container resource object has field env

Reply:
> Container resource object has field env

yes, but of env variables are bind from config map they aren't available in `env` field unfortunately.